.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish to NPM
+on:
+  push:
+    branches:
+      - master
+      - abanoubghadban/sc-65675/implement-continous-deployment-on-github
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Fetch tags
+        run: git fetch --prune --unshallow
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install dependencies 🔧
+        run: yarn install --frozen-lockfile
+      - name: Ensure that "lerna version" has been run
+        run: |
+          changed=$(yarn run --silent changed || true)
+          if [ -n "$changed" ]; then
+            echo "Error: Found changed packages. Please run 'yarn run version' before pushing changes." >&2
+            exit 1
+          fi
+      - name: Publish package on NPM 📦
+        run: yarn run publish --dist-tag experimental
```

Comment:
`--dist-tag experimental` use for testing purposes. Will be removed before merging

Reply:
removed now, it will publish to the `latest` now